apps/sriov-kernel-nse/nse.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nse
+  labels:
+    app: nse
+spec:
+  selector:
+    matchLabels:
+      app: nse
+  template:
+    metadata:
+      labels:
+        app: nse
+    spec:
+      containers:
+        - name: nse
+          image: rrandom312/cmd-nse-icmp-responder:testing
+          imagePullPolicy: Always
+          env:
+            - name: SPIFFE_ENDPOINT_SOCKET
+              value: unix:///run/spire/sockets/agent.sock
+            - name: NSE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: NSE_CONNECT_TO
+              value: unix:///var/lib/networkservicemesh/nsm.io.sock
+            - name: NSE_SERVICE_NAME
+              value: ipam
+            - name: NSE_LABELS
+              value: serviceDomain:worker.domain
+            - name: NSE_CIDR_PREFIX
+              value: 10.0.0.0/24
+          volumeMounts:
+            - name: spire-agent-socket
+              mountPath: /run/spire/sockets
+              readOnly: true
+            - name: nsm-socket
+              mountPath: /var/lib/networkservicemesh
+          # We don't actually need resources for NSE, but we need to be sure that it is going to be deployed on the
```

Comment:
How selects the correct node for the deployment?  I feel we need to cover this point in the README.md documents.

Reply:
Added comments to vfio-nsc, vfio-nse.